front/lib/api/files/upsert.ts

```diff
@@ -242,7 +274,7 @@ const upsertTableToDatasource: ProcessingFunction = async (
       name: slugify(file.fileName),
       description: "Table uploaded from file",
       truncate: true,
-      csv: content.trim(),
+      fileId: file.sId,
```

Comment:
this one is the one that will clean-up this log: https://app.datadoghq.eu/logs?query=%22%5BCSV-FILE%5D%20Received%20direct%20CSV%20not%20file%22&agg_m=count&agg_m_source=base&agg_t=count&clustering_pattern_field_path=message&cols=host%2Cservice&fromUser=true&graphType=flamegraph&messageDisplay=inline&refresh_mode=sliding&sort=time&spanID=9200759549420987463&storage=hot&stream_sort=desc&viz=stream&from_ts=1740045982658&to_ts=1740060382658&live=true

This is on the tool_ouptut and manual file upload path as we pull the content which takes precedence here even if we receive a file everywhere.